check_diff/tests/bash_commands.rs

```diff
@@ -0,0 +1,34 @@
+use check_diff::change_directory_to_path;
+use std::env;
+use std::ffi::OsStr;
+use tempfile::Builder;
+use tracing::error;
+
+#[test]
+fn cd_test() {
+    // create directory in current directory
+    let dir = Builder::new().tempdir_in("");
+
+    match dir {
+        Ok(d) => {
+            let dest_path = d.path();
+            let _ = change_directory_to_path(dest_path);
+            assert_eq!(env::current_dir().is_ok(), true);
+            assert_ne!(
+                env::current_dir().unwrap().file_name(),
+                Some(OsStr::new("check_diff"))
+            );
```

Comment:
This is the code you were referring to in https://github.com/rust-lang/rustfmt/pull/6187#issuecomment-2196426926, right? My feeling is that this isn't a necessary check and if we assert that we've correctly changed the working directory to the tempdir then I don't think this is necessary.